bin/middlewares/opensearch/os.ejs

```diff
@@ -0,0 +1,20 @@
+  </Image>
+  <Url type="text/html" method="GET" template="https://<%= baseUrl %>?q={searchTerms}&amp;client=opensearch"/>
+  <Query role="example" searchTerms="Qwant"/>
```

Comment:
That's not a very good example for Qwant Maps...

Reply:
: ]